### ISSUE TITLE
tighten up info for users with respect to GDPR law

### ISSUE THREAD

**Issue by egonw** [fnielsen/scholia#1067]
From the UM legal layers:

![image](https://user-images.githubusercontent.com/26721/76212772-81323380-6209-11ea-9763-46fc315fc2ff.png)

**Comment by fnielsen** [comment]
I suppose we can just point to Wikidata?

**Comment by egonw** [comment]
Yes, I think so. But we should likely get the info together. I am not sure where we legally stand on "collecting" data. Even the more general processing of data is unclear, as most of our processing is *enabled* by Scholia, but actually run by users in their own browsers, not by us. First step, is to just have a page that clarifies what we do, what users (browsers) do, and how people can add/remove data (and that is also not with us, but with Wikidata).

**Comment by fnielsen** [comment]
There is a bit of information at https://tools.wmflabs.org/scholia/faq What do you suggest we specifically add.

**Comment by egonw** [comment]
I need to think about this a bit. I will get back on this very soon!
